timed/tracking/serializers.py

```diff
@@ -136,6 +136,14 @@ def validate_rejected(self, value):
 
         return value
 
+    def validate_remaining_effort(self, value):
+        """Only update remaining effort when tracking is active on the corresponding project."""
+        if not self.instance.task.project.remaining_effort_tracking:
+            raise ValidationError(
+                "Remaining effort tracking is not active on this project!"
+            )
+        return value
```

Comment:
Instead of rejecting the API call, I'd just silently set it to `None` instead. No need to tell the user, as they're probably unable to do something about it

Reply:
Not so sure about that. At least they should get a ```Somehting gone wrong``` message when they can't save the remaining effort, otherwise the user might think it worked.

Reply:
Guess it depends on frontend implementation. Feel free to keep this one then